### PR TITLE
New Messaging Guide page

### PR DESCRIPTION
**Pull request by ildikov**
This patch adds a new tab that contains documentation that gives guidelines on how to talk about and advertize the project in social media and other forums.

### CODE REVIEW

**Comment by allisonprice** [comment]
@ildikov I don't think this should be part of the top level navigation. perhaps we link to it from the supporters page, like “Are you supporting StarlingX and looking for guidance on messaging?”

Then on the bottom nav, instead of adding “StarlingX out in the World” - I would just put the Messaging Guide under About. 

Let me know what you think - Allison

**Comment by ildikov** [comment]
@allisonprice Sounds good, I just re-arranged the layout the best I could. I will need help with finalizing that.

Is this look better this way to you in the preview? Is there anything in the content that you would like to change before focusing on the final layout?

**Comment by allisonprice** [comment]
@ildikov yes, i think that looks better! thanks for the updates

File: src/templates/supporters-page.js [hunk 1]
```diff
@@ -136,13 +137,13 @@ export const SupportersPageTemplate = ({
                   <a href="https://www.openinfra.dev">OpenInfra Foundation</a>{" "}
                   and a growing, global community of operators, developers and
                   organizations. Join us as we build the future of
-                  high-performance, distributed cloud infrastructure.
+                  high-performance, distributed cloud infrastructure. <a href="#messaging-guide">Interested in sharing news and information about StarlingX?</a>
```
Comment (ildikov) [inline src/templates/supporters-page.js:140]:
```suggestion
                  high-performance, distributed cloud infrastructure, and help us sharing news and information about StarlingX!
```

**Comment by ildikov** [comment]
@iamweswilson Thank you for making the changes! I looked at the current version and was wondering if we could rather turn the question that you added into a one-sentence CTA, and just have the two buttons next to each other. As that would not push the logos further down. I added a suggested edit to the text. What do you think?

**Comment by iamweswilson** [comment]
@ildikov that works for me. I'll commit your suggestion and make the button updates needed.

**Review by iamweswilson — APPROVED** [review]
@ildikov the updates are in and look good to me. You can see the latest at https://66e324fd5cbd770008bebdcd--starlingx.netlify.app/supporters/. Feel free to merge the PR when ready

**Review by allisonprice — APPROVED** [review]
Looks good to me!

**Comment by ildikov** [comment]
The content is now ready to land, and will keep evolving continuously.

**Pull request merged**